scenarios/mnist/config/query_config.json

```diff
@@ -0,0 +1,45 @@
+{
+   "datasets": [
+      {
+         "id": "19517ba8-bab8-11ed-afa1-0242ac120002",
+         "name": "mnist_1",
+         "file": "dp_mnist_1_standardised_anon.csv",
+         "select_variables": [
+
+         ],
+         "mount_path": "/mnt/remote/mnist_1/"
+      },
+      {
+         "id": "216d5cc6-bab8-11ed-afa1-0242ac120002",
+         "name": "mnist_2",
+         "file": "dp_mnist_2_standardised_anon.csv",
+         "select_variables": [
+
+         ],
+         "mount_path": "/mnt/remote/mnist_2/"
+      },
+      {
+         "id": "2830a144-bab8-11ed-afa1-0242ac120002",
+         "name": "mnist_3",
+         "file": "dp_mnist_3_standardised_anon.csv",
+         "select_variables": [
+
+         ],
+         "mount_path": "/mnt/remote/mnist_3/"
+      }
+   ],
+   "joined_dataset": {
+      "joined_dataset": "sandbox_mnist_without_key_identifiers.csv",
+      "joining_query": "",
```

Comment:
Why are these empty? It's not good practice to use empty to default to some behavior. Let's try and make the operation explicit.

Reply:
Agreed. Since the mnist scenario does not technically involve a join this was left empty. Will replace it with a mock query.

Reply:
It shouldnt be a mock query....can we have a query that does the aggregation?

Reply:
yes. Replacing it with a query which does union of all the three mnist partitions.